src/context/cart-context.js

```diff
@@ -18,8 +20,18 @@ const CartProvider = ({ children }) => {
         }
       );
       setCartProducts(response.data.cart);
+      setToastData({
+        showToast: true,
+        toastType: "success",
+        toastMesaage: "Successfully added to the cart",
+      });
     } catch (error) {
       console.log(error);
+      setToastData({
+        showToast: true,
```

Comment:
suggestion: if its true always, you can avoid setting and make it default to reduce lines and default toast type as well